### DISCUSSION TITLE
Point blank question: Continued support for HoloLens 2?

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#682] [General]
@maluoi Decision time. I have to know. In your new role, do you intend to continue supporting Microsoft's HoloLens 2 in StereoKit?

**Comment by maluoi** [comment]
I will support HoloLens 2 as long as it remains relevant enough to justify supporting it. There's still plenty of people using StereoKit with it! I currently do test desktop x64 UWP regularly, but I no longer have a HoloLens device to test on, so the ARM64 UWP build doesn't get traffic from me. Always happy to help fix issues here, but I may need extra good debug data from people to solve them.

**Reply by maluoi** [reply]
That being said, Microsoft's direction for XR since I've left has not instilled me with confidence. I don't currently recommend people to bet their businesses on Microsoft's XR hardware.

**Comment by laultman** [comment]
Fair enough answer, I can loan you a HoloLens device to test on if you will accept it. I need the testing done most efficiently as possible. I suspect that with the device you can fix things much faster than trying to dig the details out of somebody like me.
